src/NumericalAlgorithms/SphericalHarmonics/YlmSpherepack.hpp

```diff
@@ -28,23 +28,75 @@
  * Given a real-valued, scalar function \f$g(\theta, \phi)\f$, SPHEREPACK
  * expands it as:
  *
- * \f{align*}
+ * \f{align}
  * g(\theta, \phi)
  * &=\frac{1}{2}\sum_{l=0}^{l_{\max}}\bar P_l^0(\cos\theta) a_{l0}
  * +\sum_{l=1}^{l_{\max}}\sum_{m=1}^{\min(l, m_{\max})}\bar P_l^m(\cos\theta)\{
- *   a_{lm}\cos m\phi -b_{lm}\sin m\phi\}\nonumber
+ *   a_{lm}\cos m\phi -b_{lm}\sin m\phi\}\label{eq:spherepack_expansion}
  * \f}
  *
- * where \f$a_{lm}\f$ and \f$b_{lm}\f$ are spectral coefficient arrays used by
+ * where \f$a_{lm}\f$ and \f$b_{lm}\f$ are real-valued
+ * spectral coefficient arrays used by
  * SPHEREPACK, \f$P_l^m(x)\f$ are defined as
  *
- * \f{align*}
+ * \f{align}
  * \bar P_l^m(x)&=\sqrt{\frac{(2l+1)(l-m)!}{2(l+m)!}}\;P_{lm}(x)
  * \f}
  *
- * and \f$P_{nm}(x)\f$ are the associated Legendre polynomials.
+ * and \f$P_{nm}(x)\f$ are the associated Legendre polynomials as defined
+ * for example in Jackson's "Classical Electrodynamics".
  *
- * Internally, SPHEREPACK can represent such an expansion in two ways which we
+ * ##### Relationship to standard spherical harmonics
```

Comment:
Maybe only use four `####` for these headers (here and below)? When I rendered it locally, these headers had a smaller font than the actual text.